packages/peregrine/lib/store/actions/checkout/__tests__/asyncActions.spec.js

```diff
@@ -116,82 +115,6 @@ describe('resetCheckout', () => {
     });
 });
 
-describe('getCountries', () => {
```

Comment:
Add tests for `setCountries`

Reply:
Not needed anymore https://github.com/magento/pwa-studio/pull/1993/commits/68cd2b8ac59650c6a8b8a677a46d355adf1d1c0b.